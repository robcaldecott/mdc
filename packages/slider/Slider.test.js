import React from "react";
import { Slider } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <Slider
        className="custom"
        min="1"
        max="100"
        step="1"
        value="50"
        disabled
      />
    )
  ).toMatchSnapshot();
});
