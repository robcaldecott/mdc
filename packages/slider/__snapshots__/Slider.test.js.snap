// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-slider mdc-slider--disabled custom"
>
  <input
    className="mdc-range"
    disabled={true}
    max="100"
    min="1"
    step="1"
    type="range"
    value="50"
  />
</div>
`;
